holidayextras/ui-toolkit
# added basket item component

#### What does this PR do? (please provide any background)
Introduces a simple basket item component to the library.

#### What tests does this PR have?
Unit tests that can be run via `npm run test`

#### How can this be tested?
I would run `npm run test` to see unit test passing then run `npm run docs` & see it for yourself on the demo page.

#### Screenshots / Screencast
N/A

#### What gif best describes how you feel about this work?
![battles](http://i.imgur.com/IDcqKWY.gif)

#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
